java/src/main/java/org/signal/libsignal/metadata/SealedSessionCipher.java

```diff
@@ -108,43 +104,32 @@ public DecryptionResult decrypt(CertificateValidator validator, byte[] ciphertex
       byte[]      messageBytes = decrypt(staticKeys.cipherKey, staticKeys.macKey, wrapper.getEncryptedMessage());
 
       content = new UnidentifiedSenderMessageContent(messageBytes);
-      validator.validate(content.getSenderCertificate(), timestamp);
```

Comment:
This used to validate the sealed sender against the server's certificate. Since it's now self-signed we don't do that, and the validation now happens [here](https://github.com/getlantern/libsignal-metadata-java/pull/1/files#diff-88e5515dcb9fefee8f7ecebfc3d9d75d99609d0439098a42a3f322398f70caf3R56)